jans-client-api/common/src/main/java/io/jans/ca/common/response/RsCheckAccessResponse.java

```diff
@@ -15,6 +15,10 @@ public class RsCheckAccessResponse implements IOpResponse {
     private String wwwAuthenticateHeader;
     @JsonProperty(value = "ticket")
     private String ticket;
+    @JsonProperty(value = "error")
+    private String error;
+    @JsonProperty(value = "error_description")
+    private String errorDescription;
```

Comment:
Opening separate ticket to improve copy/paste of same fields
https://github.com/JanssenProject/jans/issues/1530